### ISSUE TITLE
clicking breadcrumbs loses url parameters

### ISSUE THREAD

**Issue by iros** [bocoup/mlab-vis-client#53]

**Comment by vlandham** [comment]
for this we can pass `props.location.query` to the breadcrumbs and use them in the `Link` like so:

```
  <Link to={`/location/${parent.locationKey}`} query={query}>{parent.name}</Link>
```

The problem is there are params that we might _not_ want to pass up. The specific one that we have now is `clientIsp`. Top client ISPs in a city will not be the top client ISPs in the country - for example. 

@pbeshai - does it make sense to add something to the `urlHandler` or some other portion of the URL system to extract only a subset of the params ?

`urlHandler.params(exclude=['clientIsp'])` ?

**Comment by pbeshai** [comment]
Hmm yea that's a bit tricky. I think what we should do is create a `LocationLink` component that creates the URL properly by using a subset of the props in LocationPage and the breadcrumbs can use that.

**Comment by vlandham** [comment]
As per external discussion:

Create a utility method that takes `props.location.query` and the `urlQueryConfig`. it will build a query object with only those query values that are supposed to be persisted.

**Comment by vlandham** [comment]
closed by #60
